Changes

```diff
@@ -93,23 +93,13 @@ Working version
 
 ### Standard library:
 
-* #9554: add primitive __FUNCTION__ that returns the name of the current method
-  or function, including any enclosing module or class.
-  (Nicolás Ojeda Bär, Stephen Dolan, review by Stephen Dolan)
```

Comment:
This seems to remove several existing entries from `Changes`.  Is that a mistake?

Reply:
Yes, I did a mistake when rebasing, thanks! I will fix this and rebase. (I can also change `yes` in `left`; I kept the naming scheme of `partition` to emphasize the similarities between the two implementations.))